django/db/migrations/questioner.py

```diff
@@ -162,17 +163,16 @@ def ask_not_null_alteration(self, field_name, model_name):
         """Changing a NULL field to NOT NULL."""
         if not self.dry_run:
             choice = self._choice_input(
-                "You are trying to change the nullable field '%s' on %s to non-nullable "
-                "without a default; we can't do that (the database needs something to "
-                "populate existing rows).\n"
+                "It is impossible to change a nullable field '%s' on %s to non-nullable "
+                "without providing a default because the database needs something to "
+                "populate existing rows.\n"
```

Comment:
This sentence is too long, maybe:
```python
            choice = self._choice_input(
                f"It is impossible to change a nullable field '{field_name}' "
                f"on {model_name} to non-nullable without providing a "
                f"default. This is because the database needs something to "
                f"populate existing rows.\n"
                f"Please select a fix:"
                [
                ...
```